packages/components-storybook-v2/.storybook/config.js

```diff
@@ -7,7 +7,7 @@ addParameters({
     options: {
         theme: Theme,
         showPanel: true,
-        panelPosition: 'bottom',
+        panelPosition: 'right',
```

Comment:
why this change?

Reply:
i like the addon pannel on the right side, it's more visible and clearer...